src/extension/platform.js

```diff
@@ -1,13 +1,18 @@
-import path from 'path';
```

Comment:
This function will be used for running the app directly from the directory created by the `pull-app` command.